### PR TITLE
Andriyregister

### PR DESCRIPTION
**Pull request by AndriyBilyar**
Validation done. Conflicts are fixed.

### CODE REVIEW

File: app/modules/core/views/modalView.js [hunk 1]
```diff
@@ -5,13 +5,14 @@ define(function(require) {
...
+        id: "applyModal",
 
         template: _.template(require("text!../templates/modalTemplate.html")),
 
         events:{
-        	"hidden.bs.modal #applyModal": closePopup,
-        	"click .btn-apply": submit,
-        	"click .btn-cancel": decline
+        	"hidden.bs.modal #applyModal": "closePopup",
+        	"click .btn-apply": "submit",
```
Comment (okrasn) [inline app/modules/core/views/modalView.js]: Please call method more detail, for example submitHandlerClick, etc

File: app/modules/core/views/modalView.js [hunk 2]
```diff
@@ -5,13 +5,14 @@ define(function(require) {
...
+        	"click .btn-cancel": "decline"
```
Comment (okrasn) [inline app/modules/core/views/modalView.js]: The some as prev

File: app/modules/register/template/registerTemplate.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
+
+<div class="row col-lg-8 col-lg-offset-2">
+
```
Comment (okrasn) [inline app/modules/register/template/registerTemplate.html]: Please clear white space

File: app/modules/register/template/registerTemplate.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <h5><strong> Помилка! </strong> Заповніть підсвічені поля та спробуйте знову. </h5>
+    </div>
+
+    <form action="" class="registerForm" id="registerForm">
+
+            <div class="form-group">
+                <div class="input-group name">
+                    <span class="input-group-addon"> І </span>
+                    <input type="text" class="form-control" name="name" id="name" placeholder="Ім'я">
+
```
Comment (okrasn) [inline app/modules/register/template/registerTemplate.html]: Please clear white space

File: app/modules/register/template/registerTemplate.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            <div class="form-group">
+                <div class="input-group repeatPass">
+                    <span class="input-group-addon"> * </span>
+                    <input type="password" class="form-control" id="repeatPass" name="repeatPass" placeholder="Підтвердженя паролю">
+                </div>
+            </div>                
+
+            <input type="submit" name="submit" id="submit" value="Реєстрація" class="btn btn-info center-block">
+
```
Comment (okrasn) [inline app/modules/register/template/registerTemplate.html]: Please clear white space

File: app/modules/register/view/registerView.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            },
+
+            afterRender: function () {
+                this.$el.find(".error-message").addClass("hidden");
+            },
+
+            template: _.template( require("text!../template/registerTemplate.html") ),
+
+			events: {
+				'click #submit': "submitClicked"
```
Comment (okrasn) [inline app/modules/register/view/registerView.js]: Please correct indent

File: app/modules/register/view/registerView.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+			},
+
+			submitClicked: function(e) {
+				e.preventDefault();
+
+                var feedback = {
```
Comment (okrasn) [inline app/modules/register/view/registerView.js]: Please use single var approach

File: app/modules/register/view/registerView.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            },
+
+            template: _.template( require("text!../template/registerTemplate.html") ),
+
+			events: {
+				'click #submit': "submitClicked"
+			},
+
+			submitClicked: function(e) {
+				e.preventDefault();
```
Comment (okrasn) [inline app/modules/register/view/registerView.js]: Please correct indent

File: app/modules/register/view/registerView.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+                var feedback = {
+                	name: this.$el.find('#name').val(),
+                	surname: this.$el.find('#surname').val(),
+                	email: this.$el.find('#email').val(),
+                	pass: this.$el.find('#pass').val(),
+                	repeatPass: this.$el.find('#repeatPass').val()
+                };
+
+                this.hideErrors();
+                this.model.set( feedback, {validate: true} );
```
Comment (okrasn) [inline app/modules/register/view/registerView.js]: Please prettify code

File: app/styles/scss/modules/_register-page.scss [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+            height:40px;
+        }
+
+        .input-group-addon{
+            width: 40px;
+            font-size: 116%;
+            font-weight: bold;
+        }
+
+}
```
Comment (okrasn) [inline app/styles/scss/modules/_register-page.scss]: Please correct indent

File: app/modules/register/view/registerView.js (hunk 7 above)
Comment (AndriyBilyar) [inline app/modules/register/view/registerView.js]: where should I use it?

**Pull request merged**